src/gui/editorwidgets/qgsdatetimeedit.h

```diff
@@ -63,11 +64,23 @@ class GUI_EXPORT QgsDateTimeEdit : public QDateTimeEdit
 
     /**
      * \brief dateTime returns the date time which can eventually be a null date/time
-     * \note  You mustn't call date() or time() because they can't return a NULL value.
+     * \note Before QGIS 3.10, you mustn't call date() or time() because they can't return a NULL value.
      * \note since QDateTimeEdit::dateTime() is not virtual, dateTime must be called for QgsDateTimeEdit
      */
     QDateTime dateTime() const;
 
+    /**
+     * \brief time returns the time which can eventually be a null time.
```

Comment:
eventually != éventuellement

```suggestion
     * \brief time returns the time which can be a null time.
```

Reply:
Ah thanks ! This French "false" friend! @3nids started this class 5 years ago ;-)

Going to fix them ;-)